File: src/DZarsky.TS3Viewer2.Domain/Users/Dto/UserInfoDto.cs
using DZarsky.TS3Viewer2.Domain.Users.Models;

namespace DZarsky.TS3Viewer2.Domain.Users.Dto;

public sealed class UserInfoDto
{
    public int? Id { get; set; }

    public string? Login { get; set; }

    public IList<UserRoleDto> Roles { get; set; } = new List<UserRoleDto>();

    public UserType Type { get; set; }

    public bool IsActive { get; set; }
}

File: src/DZarsky.TS3Viewer2.Web/src/api/models/UserInfoDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { UserRoleDto } from './UserRoleDto';
import type { UserType } from './UserType';

export type UserInfoDto = {
    id?: number | null;
    login?: string | null;
    roles?: Array<UserRoleDto> | null;
    type?: UserType;
    isActive?: boolean;
};